### PR TITLE
Clarification that ensure_csv uses tab separations

### LINKED ISSUE #51: loading comma-separated values format defaults to tab separators
After reading through https://en.wikipedia.org/wiki/Comma-separated_values, I think I can understand the decision behind making tab separators the default as "the only safe option," though it does seem confusing to me that `ensure_csv` assumes `sep="\t"`. Maybe worth mentioning the default `pd.to_csv()` uses commas (no tabs).

https://github.com/cthoyt/pystow/blob/2ce969069d5e5c5b86e32ce47ff2772faff1cc48/src/pystow/impl.py#L1414-L1417

### PR DESCRIPTION
**Pull request by sgbaird**
Closes #51

Feel free to modify, nix, or add as you like!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,16 @@ url = 'https://raw.githubusercontent.com/pykeen/pykeen/master/src/pykeen/dataset
 df: pd.DataFrame = pystow.ensure_csv('pykeen', 'datasets', 'nations', url=url)
 ```
 
+Ensure a comma-separated tabular data file from the internet and load it for usage (requires `pip install pandas`):
+
+```python
+import pystow
+import pandas as pd
+
+url = 'https://raw.githubusercontent.com/pykeen/pykeen/master/src/pykeen/datasets/nations/test.txt'
```
Comment (cthoyt) [inline README.md:102]: this is a tab-separated dataset... not sure this example makes sense

Comment (sgbaird) [reply README.md]: Ah, right. Added a different file.

**Review by cthoyt — APPROVED** [review]

**Comment by cthoyt** [comment]
Thanks @sgbaird :)

**Comment by sgbaird** [comment]
Sure thing! Thanks for the patience with my questions and requests 🙂

**Comment by cthoyt** [comment]
Any time, if you've got suggestions for more things in PyStow I'm happy to implement new things or accept PRs

**Pull request merged**